crates/holochain/src/core/workflow/sys_validation_workflow/validation_deps.rs

```diff
@@ -1,31 +1,41 @@
 use holo_hash::HoloHash;
 use holochain_cascade::CascadeSource;
 use holochain_types::prelude::*;
+use std::ops::Deref;
+use std::sync::Mutex;
 use std::{
     collections::{HashMap, HashSet},
     sync::Arc,
 };
 
-#[derive(Clone)]
 /// The sources of all dependencies needed in sys validation.
-/// Currently this comprises only action hashes within the same DHT, but could
-/// some day include items from other DHTs or other sources.
+#[derive(Clone)]
 pub struct SysValDeps {
     /// Dependencies found in the same DHT as the dependent
-    pub same_dht: Arc<parking_lot::Mutex<ValidationDependencies<SignedActionHashed>>>,
```

Comment:
What's the reasoning behind replacing the `parking_lot` Mutex with an `std` Mutex?